sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -19,18 +19,18 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"io"
-	"math"
-	"sort"
-	"sync"
-	"time"
-
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/graph/coder"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/sdf"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/util/ioutilx"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/internal/errors"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/log"
 	"golang.org/x/exp/maps"
+	"io"
```

Comment:
Standard Library imports should be grouped up at the top, and not with 3rd party imports (like in the pre-diff)

It's weird that these moved... Did this happen when you ran `go fmt` over the changed code?

Reply:
Yes it got changed with go fmt.